src/sentry/auth/helper.py

```diff
@@ -374,13 +375,13 @@ def _handle_unknown_identity(self, identity):
         if not op:
             if request.user.is_authenticated():
                 return self.respond('sentry/auth-confirm-link.html', {
-                    'identity': identity,
+                    'identity': InterfaceUser.to_python(identity),
```

Comment:
I see what you're trying to do here. This is a little odd, but we can talk about it in person since it's not 100% clear to me what made you do this.